src/View/WindowBarIconSetup.py

```diff
@@ -0,0 +1,21 @@
+from PySide6.QtWidgets import QWidget
+from PySide6.QtGui import QIcon, QPixmap
+
+import src.View.StyleSheetReader as StyleSheetReader
+from src.Controller.PathHandler import resource_path
+
+def setup_window(window: QWidget, title: str) -> None:
+    """
+    Protected Method for Setting up the window, with Title, StyleSheet, MinimumSize and Icon
```

Comment:
This appears to be a public method.  And in general, it's obvious from _method_name vs method_name whether it's intended to be public or private.  I'm not sure that protected is the term in Python, although it's a reasonable term given that so called private methods can be called.  Whereas in C++ or C# the distinction between private (unreachable outside the specific class itself), protected (reachable by child classes), and public (reachable by anyone) is enforced.